### ISSUE TITLE
False-positive `Can't resolve` warnings in Cargo.toml

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#7402]
## Environment

* **IntelliJ Rust plugin version:** 0.4.149.3939-211
* **Rust toolchain version:** 1.53.0 (53cb7b09b 2021-06-17) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2021.1.1 Ultimate Edition (IU-211.7142.45)
* **Operating system:** macOS 10.15.6
* **Macro expansion engine:** new
* **Name resolution engine:** new

## Problem description
If you try to use patterns in `[workspace]` table (i.e. `*` and `?` symbols), you will get false-positive `Can't resolve directory` and `Can't resolve file` warnings

<img width="445" alt="Screen Shot 2021-06-24 at 17 54 57" src="https://user-images.githubusercontent.com/2539310/123299760-87ddda80-d522-11eb-9805-e5a4e6b3bfc4.png">

## Steps to reproduce
```toml
[workspace]
members = [
    "lesson1/*/",
]
exclude = [
    "**/*.yaml"
]
```

**Comment by Kobzol** [comment]
@Undin In theory, we could expand the glob in the inspection, but I worry about possible edge cases and DDOS attacks :D If the glob was nasty. So I propose to just disable the inspection if the path contains a wildcard `*`?
